app/templates/components/donations/donation-progress.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="information">
+  <div class="amount">
+    <div class="value">{{format-currency amountDonated trimZero=true}}</div>
+    <div class="descriptive-text">given each month</div>
+  </div>
+  <div class="percentage">
+    <div class="value">{{format-percentage percentage trimZero=true}}</div>
+    <div class="descriptive-text">of goal</div>
+  </div>
+</div>
+{{progress-bar-container percentage=percentage}}
+<div class="description">{{donationGoal.description}}</div>
```

Comment:
We're using what I think are too many classes here. Are you able to rely on other primitives like HTML5 tags that are good candidates here?